Extensions/AdvancedP2PEventHandling-header.json

```diff
@@ -0,0 +1,15 @@
+{
+  "shortDescription": "Adds a way to handle events the advanced way. It is more complex but also potentially more performant.",
+  "extensionNamespace": "",
+  "fullName": "Advanced p2p event handling",
+  "name": "AdvancedP2PEventHandling",
+  "version": "1.0.0",
+  "url": "Extensions/AdvancedP2PEventHandling.json",
+  "headerUrl": "Extensions/AdvancedP2PEventHandling-header.json",
+  "tags": "p2p,performance",
```

Comment:
let's added `advanced`, so we can maybe later differentiate these extensions in another category :)